app/controllers/ncr/work_orders_controller.rb

```diff
@@ -4,12 +4,11 @@ class WorkOrdersController < UseCaseController
     MAX_UPLOADS_ON_NEW = 10
 
     def new
-      @approver_email = self.suggested_approver_email
+      @model_instance.approving_official_email = self.suggested_approver_email
```

Comment:
I tend to use `self` for class methods only - thoughts on just using `suggested_approver_email` here?